java/com/google/re2j/MakeJavaCategories.java

```diff
@@ -0,0 +1,90 @@
+package com.google.re2j;
```

Comment:
Your files need a license on top.  You can look at any of the checked in code for what the license should be.